client/pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
   "pyaes~=1.6",
   "rsa~=4.9",
   "markdown-it-py~=3.0",
+  "typing-extensions~=4.12.2",
```

Comment:
Can you explain why we need this? I'm fine with increasing the minimum-supported-Python-version if needed instead.

Reply:
This was for [supported python versions](https://devguide.python.org/versions/#supported-versions) that do not support Self